apps/prairielearn/src/lib/db-types.ts

```diff
@@ -391,3 +391,92 @@ export const AssessmentQuestionSchema = z.object({
   tries_per_variant: z.number().nullable(),
 });
 export type AssessmentQuestion = z.infer<typeof AssessmentQuestionSchema>;
+
+export const RubricSchema = z.object({
+  created_at: DateFromISOString,
+  deleted_at: DateFromISOString.nullable(),
+  id: IdSchema,
+  max_extra_points: z.number(),
+  min_points: z.number(),
+  modified_at: DateFromISOString,
+  replace_auto_points: z.boolean(),
+  starting_points: z.number(),
+});
+export type Rubric = z.infer<typeof RubricSchema>;
+
+export const RubricItemSchema = z.object({
+  always_show_to_students: z.boolean(),
+  deleted_at: DateFromISOString.nullable(),
+  description: z.string().max(100),
+  explanation: z.string().max(10000).nullable(),
+  grader_note: z.string().max(10000).nullable(),
+  id: IdSchema,
+  key_binding: z.string().max(30).nullable(),
+  number: IdSchema,
```

Comment:
This is a little strange. Should `IdSchema` be `BigIntSchema` instead? 🤔 

For my own knowledge: why is `number` a bigint anyways?

Reply:
Yeah, number doesn't need to be a bigint. Want me to change the type with a migration?

Reply:
Yeah, I think that's a good idea, it makes downstream code simpler when you can treat a number as an actual number. I checked in prod and we only have ~2.5k rubric items at the moment, so  we should do that before the table gets much larger since Postgres will have to rewrite the entire table. Feel free to do that in another PR first.